test/generateBaseline.sh

```diff
@@ -24,8 +24,48 @@ WORKSPACE=`cd ${TEST_DIR}/..; cd -`
 TEMP_DIR=`mktemp -d`
 APP_NAME="slowFixDecoder"
 
-${TEST_DIR}/tarFiles.sh -d ${TEMP_DIR} -a ${APP_NAME} -x
+function usage() {
+cat << EOM
+-------------------------------------------------------------------------------------
+usage: ${PROGRAM_NAME} -h
+-h|--hostname   remote host to generate files on (host must have performance counters enabled
 
 python ${PYTEST_DIR}/test_xpedite/test_profiler/generateBaseline.py ${TEMP_DIR} ${APP_NAME} ${WORKSPACE}
+-------------------------------------------------------------------------------------
```

Comment:
The python command above doesn't look right. Looks like left over code from merge. Can you please review for merge issues ?

Reply:
This is just usage, should I remove this?